habitat_sim/simulator.py

```diff
@@ -285,6 +285,9 @@ def apply_force(self, force, relative_position, object_id, scene_id=0):
     def apply_torque(self, torque, object_id, scene_id=0):
         self._sim.apply_torque(torque, object_id, scene_id)
 
+    def contact_test(self, object_id, scene_id=0):
```

Comment:
It it is an internal test, I am wonderingwhy we need to expose it on the python side?

I took a look at the other bindings above it, e.g., apply_torque, get_rotation, add_object. They are reasonable to be exposed here. But this one is different. I would like to hear from you first.

Reply:
Good question. This function name may be misleading. It is not a unit test, but a discrete collision check. I'm happy to rename it if that is a problem. 

There are actually a couple of applications for this functionality on the python side. 

Examples include:
 - 3D step filter function (check if agent collides with env after moving)
 - Scene generation and 3D Agent placement (rejection sampling for contact free placement)